shell/src/main/java/alluxio/cli/fs/command/DistributedLoadUtils.java

```diff
@@ -88,8 +89,15 @@ private static void load(AbstractDistributedJobCommand command, List<URIStatus>
       Set<String> excludedWorkerSet, Set<String> localityIds, Set<String> excludedLocalityIds,
       boolean printOut) throws IOException, AlluxioException {
     ListStatusPOptions options = ListStatusPOptions.newBuilder().setRecursive(true).build();
+    AtomicInteger incompleteCount = new AtomicInteger();
```

Comment:
Why we use AtomicInteger here?

Reply:
Variable used in lambda expression should be final or effectively final, so we need to use AtomicInteger object.

Reply:
seems like we are not doing this in parallel, so it might not be necessary to use AtomicInteger? even if we were to use concurrent data structure, checkout LongAdder, it is much more efficient for increment operations.

Reply:
@yuzhu I replaced AtomicInteger with LongAdder,PTAL,thx!